src/Server.php

```diff
@@ -386,6 +386,10 @@ public function getMotd() : string{
 		return $this->configGroup->getConfigString("motd", self::DEFAULT_SERVER_NAME);
 	}
 
+	public function getSubMotd() : string{
```

Comment:
A doc comment should be added that explains where this is seen (i.e. only on LAN servers, below the main server name).

Reply:
I can add

Reply:
The doc comment you added gives no more useful information than the function name itself.